admin/base/views.py

```diff
@@ -5,14 +5,13 @@
 from django.utils.http import urlsafe_base64_decode
 from admin.common_auth.models import MyUser
 
-
 @login_required
 def home(request):
     context = {}
     return render(request, 'home.html', context)
 
 def password_reset_done(request, **kwargs):
-    messages.success(request, 'You have successfully reset your password and activated your admin account. Thank you')
+    messages.success(request, 'You have successfully reset your password and activated your admin account.')
```

Comment:
Change password is when a user is already logged in and wants to change their password. The hoops they should have to jump through should just be password length and repetition.

Reset password is when a user forgets their login information and they need a link or temporary password sent to their Email.

So if they are activating their account it shouldn't look as though they had to reset the password, and if they are resetting their password it shouldn't look like they are activating their account.